test/integration/scheduler/priorities_test.go

```diff
@@ -36,7 +39,29 @@ import (
 // TestNodeAffinity verifies that scheduler's node affinity priority function
 // works correctly.
 func TestNodeAffinity(t *testing.T) {
-	testCtx := initTest(t, "node-affinity")
+	prof := schedulerconfig.KubeSchedulerProfile{
+		SchedulerName: v1.DefaultSchedulerName,
+		Plugins: &schedulerconfig.Plugins{
+			Score: &schedulerconfig.PluginSet{
+				Enabled: []schedulerconfig.Plugin{
+					{Name: nodeaffinity.Name, Weight: 1},
+				},
+				Disabled: []schedulerconfig.Plugin{
+					{Name: "*"},
+				},
+			},
+		},
+	}
+	testCtx := testutils.InitTestSchedulerWithOptions(
+		t,
+		testutils.InitTestMaster(t, "node-affinity", nil),
+		false,
+		nil,
+		0,
+		scheduler.WithProfiles(prof),
+	)
+	testutils.SyncInformerFactory(testCtx)
+	go testCtx.Scheduler.Run(testCtx.Ctx)
```

Comment:
I expect all other score tests have the same issue, can we wrap this around a function and use it everywhere?:

```
func initTestWithPlugins(name string, enabledPlugins []schedulerconfig.Plugin)  *TestContext {
        prof := schedulerconfig.KubeSchedulerProfile{
		SchedulerName: v1.DefaultSchedulerName,
		Plugins: &schedulerconfig.Plugins{
			Score: &schedulerconfig.PluginSet{
				Enabled:enabledPlugins,
				Disabled: []schedulerconfig.Plugin{
					{Name: "*"},
				},
			},
		},
	}
	testCtx := testutils.InitTestSchedulerWithOptions(
		t,
		testutils.InitTestMaster(t, name, nil),
		false,
		nil,
		0,
		scheduler.WithProfiles(prof),
	)
	testutils.SyncInformerFactory(testCtx)
	go testCtx.Scheduler.Run(testCtx.Ctx)
        return testCtx
}
```

Reply:
sure, will update.